### ISSUE TITLE
Row headers missing if title is hidden

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/boardscribe#63]
The meeting title has a `rowheader` role added:

<img width="1547" height="638" alt="Image" src="https://github.com/user-attachments/assets/eea147c6-d60d-4cbd-a7a5-a97059084eb3" />

```
<tr role="row"><td data-label="Title" role="rowheader">Regular Meeting</td><td data-label="Date" role="cell">Wednesday, April 21, 2027</td><td data-label="Agenda" role="cell"><span class="sr-text screen-reader-text">Agenda not available</span></td><td data-label="Minutes" role="cell"><span class="sr-text screen-reader-text">Minutes not available</span></td></tr>
```

If the title column is hidden, then there is no row header:

<img width="1583" height="406" alt="Image" src="https://github.com/user-attachments/assets/013d6bc1-16fe-4658-a0a8-adbfc4ec0d57" />

```
<tr role="row"><td data-label="Title" role="rowheader">Regular Meeting</td><td data-label="Date" role="cell">Wednesday, April 21, 2027</td><td data-label="Agenda" role="cell"><span class="sr-text screen-reader-text">Agenda not available</span></td><td data-label="Minutes" role="cell"><span class="sr-text screen-reader-text">Minutes not available</span></td></tr>
```

If the title is hidden, the date should have `role="rowheader"` to ensure there is a row header present.

**Comment by SteveJonesDev** [comment]
Title row header:
<img width="1436" height="363" alt="Image" src="https://github.com/user-attachments/assets/7acde9e1-0a00-4d9d-9f96-53163613d4f7" />

Date row header:
<img width="1389" height="187" alt="Image" src="https://github.com/user-attachments/assets/0b3e32b8-56d3-4ca7-86a9-2287429928c4" />

If not title or date, then no `rowheader` is added:
<img width="1386" height="250" alt="Image" src="https://github.com/user-attachments/assets/bef7ff55-8a36-4f8a-bed8-7aa4df88f8f3" />

@amberhinds  playground link here for testing: https://github.com/equalizedigital/boardscribe/pull/64#issuecomment-5257178603

**Comment by amberhinds** [comment]
I tested this and it look good. Thank you!
